src/devices/cpu/e132xs/e132xsop.hxx

```diff
@@ -1311,191 +1250,248 @@ void hyperstone_device::hyperstone_add_global_global()
 	m_icount -= m_clock_cycles_1;
 }
 
-void hyperstone_device::hyperstone_add_global_local()
+template <hyperstone_device::reg_bank DST_GLOBAL, hyperstone_device::reg_bank SRC_GLOBAL>
+void hyperstone_device::hyperstone_adds()
 {
-	check_delay_PC();
-
-	const uint32_t dst_code = DST_CODE;
-
-	const uint32_t sreg = m_local_regs[(SRC_CODE + GET_FP) & 0x3f];
-	uint32_t dreg = m_global_regs[dst_code];
-
-	const uint64_t tmp = (uint64_t)sreg + (uint64_t)dreg;
+	const uint32_t fp = GET_FP;
```

Comment:
There's no call to `check_delay_PC` here - the similar `negs` handler does have such a call. Is this intentional or an oversight?